### ISSUE TITLE
Firefox 57: NVDA unable to Focus on Downloads List

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#7779]
With firefox 57, I've noticed that NVDA is not able to focus on the Downloads list unless Object Navigation is first used to move the mouse. I'm aware that there are several separate issues, but wanted to point this one out.

**Comment by ehollig** [comment]
With Firefox 57 and the latest Master of NVDA, I am able to focus on the downloads list after pressing CTRL+J to bring up the list of downloads, then tab and shift+tab

**Comment by Adriani90** [comment]
I can focus the downloads list in NVDA 2017.3 and Firefox 57 after pressing ctrl + j. Actually it brings me directly into the downloads list.

**Comment by fernando-jose-silva** [comment]
For me, too, nvda does not read the focus in the list by pressing ctrl + j.
To read the download list you need to press tab and then shift + tab.
firefox 57 64bits and nvda next 14617

**Comment by derekriemer** [comment]
This is probably a firefox issue. @jcsteh is there an open Mozilla ticket anywhere you've by chance made, or should I create one under firefox?

**Comment by jcsteh** [comment]
I've seen this too. I'm not aware of any open bug. If you do file one, please file it under Firefox and CC me. Thanks.

**Comment by mohdshara** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1424469 has been just reported. @jcteh is mentioned.

**Comment by jcsteh** [comment]
Fixed for Firefox 60. Hopefully, it will also be uplifted to Firefox 59.
